### DISCUSSION TITLE
Internal server error on https://git.meli.delivery/meli/meli/issues/

### DISCUSSION THREAD

**Discussion by oblitum** [meli/meli#14] [General]
I was trying to comment on issues at https://git.meli.delivery/meli/meli/issues/ but it gives a big 500 error after trying to send the comment.

**Comment by epilys** [comment]
Thanks for the report, didn't notice. I moved it to a new server and I didn't set up error monitoring yet.

The error is either a gitea bug or corrupted database bug:

> 2022/08/09 19:34:58 models/issue_xref.go:294:RefCommentHTMLURL() [E] LoadRefComment(0): comment does not exist [id: 0, issue_id: 0]
2022/08/09 19:35:12 models/issue_xref.go:294:RefCommentHTMLURL() [E] LoadRefComment(0): comment does not exist [id: 0, issue_id: 0]
2022/08/09 19:35:49 ...outers/repo/issue.go:2079:NewComment() [E] CreateIssueComment: NOT NULL constraint failed: comment.id 
2022/08/09 19:36:06 ...outers/repo/issue.go:1068:ViewIssue() [E] GetIssueByIndex: issue does not exist [id: 0, repo_id: 17, index: 102]

Will check the database for validity and report back

**Reply by epilys** [reply]
@oblitum  Indeed it was a corrupted database schema, I've restored it and everything seems to work again.

**Reply by oblitum** [reply]
@epilys okay, thx!
